eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/JsonNodeToClusteredVariantProcessor.java

```diff
@@ -34,11 +34,12 @@
 /**
  * Processes a DbSNP JsonNode to produce a clustered variant entity.
  */
-public class JsonNodeToClusteredVariantProcessor implements ItemProcessor<JsonNode, DbsnpClusteredVariantEntity> {
+public class JsonNodeToClusteredVariantProcessor implements ItemProcessor<JsonNode, ClusteredVariantEntity> {
```

Comment:
We are importing variants from dbSNP, so this should still be a `DbsnpClusteredVariantEntity`.